packaging/osx/app/Info.plist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>CFBundleDisplayName</key>
+        <string>Datadog Agent</string>
+        <key>CFBundleExecutable</key>
+        <string>gui</string>
+        <key>CFBundleIconFile</key>
+        <string>Agent.icns</string>
+    	<key>CFBundleGetInfoString</key>
+    	<string>5.2, Copyright © 2009-2015, Datadog Inc.</string>
+        <key>CFBundlePackageType</key>
+        <string>APPL</string>
+        <key>CFBundleVersion</key>
+        <string>5.2.0</string>
```

Comment:
Nope, it's updated there during the build https://github.com/DataDog/dd-agent/pull/1361/files#diff-2eeaed663bd0d25b7e608891384b7298R130
(I should probably remove this default though, it was only for tests)